### DISCUSSION TITLE
Which spades subcommand to use for full synthetic promoters?

### DISCUSSION THREAD

**Discussion by biomystery** [ablab/spades#1091] [Q&A]
I have a special case: 

* We've made a library of synthetic promoters using a set of short motifs (~15bp average, n=20) by random ligation. 
*  The length of resultant synthetic promoters is within a range from  250bp~1000bp. 
* In order to get longer promoter info, we did sonication and performed 2x300bp sequencing
* Now, we need to use the 2x300bp reads to recover the library of synthetic promoters (seqs + abundance) 

Questions @asl :
* Can we apply Spade here? Is the assembly graph includes all possible promoter sequences in our library? 
* Which sub-tools from Spade to use? Can we use `biosyntheticSPAdes` to annotate the promoter sequences? 

Thank you so much!

**Comment by asl** [comment]
So, how long are your fragments overall?

**Comment by biomystery** [comment]
@asl  The range should be 200~600bp post sonification.           before sonication is 250\~1000
